wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientTableViewCell.swift

```diff
@@ -119,79 +93,59 @@ class ClientTableViewCell: UITableViewCell, DynamicTypeCapable {
 
-            fingerprintLabel.topAnchor.constraint(equalTo: labelLabel.bottomAnchor, constant: 4),
-            fingerprintLabel.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 16),
-            fingerprintLabel.rightAnchor.constraint(lessThanOrEqualTo: contentView.rightAnchor, constant: -16),
-            fingerprintLabel.heightAnchor.constraint(equalToConstant: 16),
+            proteusVerficiationStatusImageView.topAnchor.constraint(equalTo: nameLabel.topAnchor),
+            proteusVerficiationStatusImageView.leftAnchor.constraint(equalTo: nameLabel.rightAnchor, constant: 8),
+            proteusVerficiationStatusImageView.heightAnchor.constraint(equalToConstant: 16),
+            proteusVerficiationStatusImageView.widthAnchor.constraint(equalToConstant: 16),
 
-            activationLabel.topAnchor.constraint(equalTo: fingerprintLabel.bottomAnchor, constant: 8),
-            activationLabel.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 16),
-            activationLabel.rightAnchor.constraint(lessThanOrEqualTo: contentView.rightAnchor, constant: -16),
+            mlsThumbprintLabel.topAnchor.constraint(equalTo: labelLabel.bottomAnchor, constant: 4),
+            mlsThumbprintLabel.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 16),
+            mlsThumbprintLabel.rightAnchor.constraint(lessThanOrEqualTo: contentView.rightAnchor, constant: -16),
 
-            verifiedLabel.topAnchor.constraint(equalTo: activationLabel.bottomAnchor, constant: 4),
-            verifiedLabel.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 16),
-            verifiedLabel.rightAnchor.constraint(lessThanOrEqualTo: contentView.rightAnchor, constant: -16),
-            verifiedLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -16)
-        ])
-    }
+            proteusIdLabel.topAnchor.constraint(equalTo: mlsThumbprintLabel.bottomAnchor, constant: 0),
+            proteusIdLabel.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 16),
+            proteusIdLabel.rightAnchor.constraint(lessThanOrEqualTo: contentView.rightAnchor, constant: -16),
+            proteusIdLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -16),
 
-    private func updateVerifiedLabel() {
-        if let userClient = userClient,
-           showVerified {
-
-            if userClient.verified {
-                verifiedLabel.text = L10n.Localizable.Device.verified.capitalized
-            } else {
-                verifiedLabel.text = L10n.Localizable.Device.notVerified.capitalized
-            }
-        } else {
-            verifiedLabel.text = ""
-        }
-    }
+            labelLabel.topAnchor.constraint(equalTo: nameLabel.bottomAnchor, constant: 2),
+            labelLabel.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 16),
+            labelLabel.rightAnchor.constraint(lessThanOrEqualTo: contentView.rightAnchor, constant: -16)
```

Comment:
suggestion: Even though it's out of the scope for this PR, can we use an enum and put all of those constants inside the enum so we can avoid using magic numbers?

Reply:
May be we will do a clean up later.